src/Microsoft.VisualStudio.ProjectSystem.Managed/ImmutableCollectionLinqExtensions.cs

```diff
@@ -39,6 +39,14 @@ internal static class ImmutableCollectionLinqExtensions
             return false;
         }
 
+        public static IEnumerable<TResult> Select<TSource, TResult>(this ImmutableArray<TSource> immutableArray, Func<TSource, TResult> selector)
```

Comment:
Doesn't this already exist?https://github.com/dotnet/corefx/blob/master/src/System.Collections.Immutable/src/System/Linq/ImmutableArrayExtensions.cs#L30

Did it not get added to the version that we use?

Reply:
You're right. I missed that.

I checked the other extensions we have and they're not duplicates.